app/helpers/checkouts_helper.rb

```diff
@@ -20,10 +20,18 @@ def render_checkout_due_date(checkout)
   end
 
   def render_renewal_select(checkout)
-    check_box_tag 'renewal_list[]', checkout.item_key, false, class: 'checkbox', multiple: true
+    check_box_tag 'renewal_list[]', checkout.item_key, false,
```

Comment:
🤔 wonder if it would make sense to combine functionality with `HoldsHelper::render_hold_select` (or at least part of it). Might be overkill just to avoid repeating a single line of code.

Reply:
I was thinking about a similar thing when working on these, like select all as a component 🤔

Reply:
Oh man. Love that idea. I think for now lets leave as is and chat in Slack about making a new component.